ios/ImagePickerManager.m

```diff
@@ -331,32 +338,25 @@ - (void)imagePickerController:(UIImagePickerController *)picker didFinishPicking
 {
     dispatch_block_t dismissCompletionBlock = ^{
         NSMutableArray<NSDictionary *> *assets = [[NSMutableArray alloc] initWithCapacity:1];
+        PHAsset *asset = nil;
+
+        // If include extra, we fetch the PHAsset, this required library permissions
+        if([self.options[@"includeExtra"] boolValue]) {
+          asset = [ImagePickerUtils fetchPHAsset:info];
```

Comment:
Can we catch an error and report it if permissions haven't been granted?

Reply:
ah, good shout! could we do this as a follow-up p/r as this logic of fetching the asset has already been merged/released?

Reply:
@Johan-dutoit I'm pretty sure the library returns an error further up the chain if permissions are NOT granted but you passed `includeExtra`, but I agreed with @helenaford this should be done in a separate PR.

Maybe we could enable project boards on this repo @Johan-dutoit and start adding some improvement ideas there?

Reply:
Agree on both points.